site/pages/home/index.js

```diff
@@ -5,16 +5,14 @@ import CaseStudy from '../../components/case-study';
 import Brie from './brie-slice';
 import TechSlice from '../../slices/tech-slice';
 import BlogSlice from './blog-slice';
-import ContactUs from './contact-us-slice';
 import NewsLetter from './newsletter-slice';
 
-const HomePage = ({ contactUsURL, featuredBlogPosts }) => (
+const HomePage = ({ featuredBlogPosts }) => (
   <div>
     <TopSlice />
     <CaseStudy />
     <Brie />
     <TechSlice />
-    <ContactUs postURL={contactUsURL} />
```

Comment:
Do you mean that you want the contact us section on both home page and about us page?